data/data/bootstrap/baremetal/files/usr/local/bin/master-bmh-update.sh

```diff
@@ -35,7 +35,7 @@ echo "Stopping provisioning services..."
 systemctl stop ironic.service
 
 echo "Wait for control plane to fail over"
-sleep 30
+sleep 120
```

Comment:
Since what we're worried about is that we don't have the bootstrap BMO operating on the workers, maybe instead of a hardcoded timeout we can do something like:

```
systemctl --no-block stop ironic.service
while systemctl is-running metal3-baremetal-operator.service; do
    sleep 10
done
```

Reply:
Added